src/editor.cc

```diff
@@ -7,8 +7,8 @@ void EditorInterface::wheelEvent(QWheelEvent *event)
 	QSettingsCached settings;
 	bool wheelzoom_enabled = Preferences::inst()->getValue("editor/ctrlmousewheelzoom").toBool();
 	if ((event->modifiers() == Qt::ControlModifier) && wheelzoom_enabled) {
-		if (event->delta() > 0) zoomIn();
-		else if (event->delta() < 0) zoomOut();
+		if (event->angleDelta().y() > 0) zoomIn();
+		else if (event->angleDelta().y() < 0) zoomOut();
```

Comment:
`angleDelta` was preferred here as the [documentation claims there's an issue](https://doc.qt.io/qt-5/qwheelevent.html#pixelDelta) with `pixelDelta` on X11.